package.json

```diff
@@ -124,6 +128,7 @@
     "import-local": "^2.0.0",
     "interpret": "^1.1.0",
     "loader-utils": "^1.1.0",
+    "prettier": "^1.17.0",
```

Comment:
@pranshuchittora was there a reason `prettier` needed to be added to `dependencies` instead of `devDependencies`?

Reply:
Fix submitted as #968.